src/read_write/ply.rs

```diff
@@ -484,27 +484,28 @@ impl PlyNodeWriter {
         }
     }
 
-    fn create_header(&mut self, layers: &[NodeLayer]) {
+    fn create_header(&mut self, attributes: &[&str]) {
```

Comment:
Wouldn't we want an attribute that is essentially (&str, attr_type) here? Otherwise this only works for known attributes.

Reply:
could the attribute type be specified in the future from the node meta data?

Reply:
@nnmm Yes, you are right, once the `write` function works over `PointData` instead of `Point`, we want this. I am currently addressing this in a follow-up PR.
@catevita Yes, once we have metadata, we can utilize it, but still the writers should not know too much about metadata, the one calling them probably should...

Reply:
yes, I agree